sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreCache.java

```diff
@@ -41,13 +41,18 @@
 
     private static StateStoreCache instance = null;
 
+    @VisibleForTesting
     protected final StateStore store;
 
+    @VisibleForTesting
```

Comment:
Just annotating that these are only non-private so that tests can use them (specifically by `StateStoreCacheTest.TestStateStoreCache.consistencyCheckForTests()`)